src/components/header/header.module.css

```diff
@@ -1,9 +1,10 @@
 .container {
   display: flex;
-  justify-content: space-between;
-  padding: 0 var(--size-4) var(--size-2);
-  margin: 0;
+  justify-content: center;
```

Comment:
The padding / margin removal has made things a bit more cramped at the bottom of the header
![Screenshot 2024-10-18 at 12 44 08 PM](https://github.com/user-attachments/assets/915daa62-e88d-4618-9bf1-3cbf966fbaec)

Made [one small tweak](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/pull/113/commits/a0787c6bf7a4c9fa55e6f14225a2d103b089adb4) to retain some of that and now I think we're good to go!
![Screenshot 2024-10-18 at 12 44 13 PM](https://github.com/user-attachments/assets/9065bf01-752f-4730-9623-5e02b832dbad)